Annotato/Annotato/Views/Document/List/DocumentListImportMenu.swift

```diff
@@ -70,4 +75,12 @@ class DocumentListImportMenu: UIStackView {
         actionDelegate?.didTapFromCodeButton()
         self.isHidden = true
     }
+
+    private func setUpSubscribers() {
+        NetworkMonitor.shared.$isConnected.sink(receiveValue: { [weak self] isConnected in
+            DispatchQueue.main.async {
+                self?.fromCodeButton.isEnabled = isConnected
+            }
+        }).store(in: &cancellables)
+    }
```

Comment:
This seems to work for me quite well actually. Why did you say that the isConnected thing is not working properly? How to reproduce?

Reply:
I enter the app with wifi connected, then I disconnect the wifi, up to here is correct. But when I turn on the wifi again, `isConnected` doesn't becomes true (it logs as false).